static/js/statsvar_menu.tsx

```diff
@@ -80,15 +79,15 @@ class Node extends Component<NodePropType, NodeStateType> {
             ></Node>
           );
         });
+      } else {
+        expandImg = (
+          <img
+            className="right-caret"
+            src="../images/right-caret.png"
```

Comment:
try look at the changes in this PR: https://github.com/datacommonsorg/website/commit/e8c3fa6526c57c3a2d2fcdf2856384a902ad8846#diff-08726cc9ace23446d1e230987e2f5ce6R119-R121

and see if you need change accordingly

Reply:
Resolved the change in right-caret. 
But there're two lines on the left of the menu, https://drive.google.com/file/d/1n_DugJUTgzMMQLR_DrXsq9A6M7ldlzWr/view?usp=sharing
I think I need help from @beets to get rid of it.